examples/api/lib/widgets/heroes/hero.1.dart

```diff
@@ -18,9 +18,8 @@ class HeroApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      theme: ThemeData(useMaterial3: true),
```

Comment:
I noticed this and in some other example are removing this flag, can you explain a bit why we want to remove it?

Reply:
Ah we manually set `useMaterial3` to true when its default value was false. The default value becomes to true since last year so `useMaterial3: true` is redundant.

Reply:
thanks for explanation. SGTM